ionic_user_interface/src/components/UserRouteList.vue

```diff
@@ -8,13 +8,21 @@
     >
       <ion-card-header>
         <ion-card-title>{{ route.routeName }}</ion-card-title>
-        <VoteButton
-          class="vote-button"
-          :username="route.username"
-          :createdAt="route.createdAt"
-          v-model:voteCount="route.voteCount"
-          v-model:hasVoted="route.hasVoted"
-        ></VoteButton>
+        <div v-if="isOwnself || isAdmin" class="center-right">
```

Comment:
Both `delete` button and `vote` button do not show up if not logged in. I think the better way is still show the `vote` when not logged in.

Reply:
Done

Reply:
Minor issue is that clicking on the Vote button will do nothing when logged in (won't even direct them to the viewRoute page), but it should be a negligible inconvenience